tools/scenario-player/scenario_player/tasks/blockchain.py

```diff
@@ -114,7 +114,7 @@ def _run(self, *args, **kwargs):
             web3=self.web3,
             contract_manager=self._runner.contract_manager,
             contract_address=self._runner.token_network_address,
-            contract_name=CONTRACT_TOKEN_NETWORK,
+            contract_name=self.contract_name,
```

Comment:
Can we be sure `self.contract_name` is set here? Maybe the default should be changed from `None` to `CONTRACT_TOKEN_NETWORK`?

Reply:
In line 101 it is checked that all keys exist. I'd say that this in enough.